tests/test_debugging.py

```diff
@@ -364,6 +364,10 @@ def circuit():
     @pytest.mark.parametrize("diff_method", [None, "parameter-shift"])
     def test_default_qutrit_mixed_finite_shot(self, diff_method):
         """Test that multiple snapshots are returned correctly on the qutrit density-matrix simulator."""
+
+        # TODO: not sure what to do with this test so leaving this here for now.
+        np.random.seed(9872653)
```

Comment:
I find that it is problematic that this test depends on this global seed to pass